lib/hope-tab.js

```diff
@@ -243,14 +245,6 @@ window.HopeTab = fxosComponent.register('hope-tab', {
         box-sizing: content-box;
       }
 
-      .url .field .reader {
-        background-image: url(assets/urlbar_reader_mode_icon.png);
-      }
-
-      .url .field .refresh {
-        background-image: url(assets/urlbar_refresh_icon.png);
```

Comment:
If UX want these specific icons I think we need to get them added to fxos-icons (or perhaps a 'next' fork) before we land this change :)